code/game/machinery/computer/computer.dm

```diff
@@ -140,3 +140,60 @@
 /obj/structure/machinery/computer/fixer/New()
 	all_configs = config
 	..()
+
+/obj/structure/red_terminal
```

Comment:
if it's not using parent /obj/structure/machinery/computer
it should probably be in it's own file.

Reply:
for one-off event PRs, i don't really care